src/components/Wishlist/Wishlist.js

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from 'react';
+import {
+  Accordion,
+  AccordionSummary,
+  AccordionDetails,
+  Typography,
+  FormGroup,
+  FormControlLabel,
+  Checkbox,
+  IconButton,
+} from '@material-ui/core/';
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore';
+import DeleteIcon from '@material-ui/icons/Delete';
```

Comment:
Good idea to bring in a tech like Material UI here for a project like this one - it can save you a lot of time and energy in making things look the way you want them to!